.gitignore

```diff
@@ -12,5 +12,7 @@ wss-unified-agent.jar
 whitesource/
 *.swp
 
+data1.txt.gz
```

Comment:
do we need it here? where was it created?

Reply:
This file is created during old tests and should be ignored by git.

Reply:
how is it related with the docs?